### PR TITLE
move images to country specific folders, in order to support differen…

### LINKED ISSUE #182: Add support for country specific image-folders

### PR DESCRIPTION
**Pull request by fossecode**
…t logos and social media images. Fixes #182

### CODE REVIEW

**Comment by fossecode** [comment]
Any thoughts on this solution regarding images? Ok or too brittle?

File: config.example.json [hunk 1]
```diff
@@ -1 +1,4 @@
-{}
+{
+  "BASE_URL": "https://coronastatus.no",
+  "IMAGE_SUBFOLDER": "no"
```
Comment (michaelmcmillan) [inline config.example.json]: Maybe have a more general setting that we can use for multiple things, `LANGUAGE` for instance?

Comment (fossecode) [reply config.example.json]: Yup, sounds good!

Comment (fossecode) [reply config.example.json]: Changed now

File: app/server.ts [hunk 2]
```diff
@@ -9,14 +9,15 @@ import apiRoutes from './routes/api-routes';
...
+import { LANGUAGE, BASE_URL } from '../config.json';
 
 const app = express();
 const port = process.env.PORT || 7272;
 const isDevelopmentEnv = process.env.NODE_ENV === 'dev';
 
 i18n.configure({
   locales: ['en', 'no', 'nl'],
-  defaultLocale: 'en',
+  defaultLocale: LANGUAGE,
```
Comment (fossecode) [inline app/server.ts:20]: Fallback language is now specified here.

**Review by michaelmcmillan — APPROVED** [review]

**Review by adriaandotcom — APPROVED** [review]

**Comment by adriaandotcom** [comment]
I think this solution is fine for now.

**Pull request merged**